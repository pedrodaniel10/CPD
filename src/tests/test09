4 20 100000 5