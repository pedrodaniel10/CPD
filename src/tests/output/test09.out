0.92 0.14
0.50 0.50
